tc/sgx/trusted_worker_manager/enclave/work_order_processor.h

```diff
@@ -26,14 +26,15 @@
 namespace tcf {
         class WorkOrderProcessor {
         public:
-                WorkOrderProcessor() {}
-                ~WorkOrderProcessor() {
-                    // Sanitize class members storing secrets
-                    worker_encryption_key.clear();
-                    session_key.clear();
-                }
-                ByteArray CreateErrorResponse(int err_code, const char* err_message);
-                ByteArray Process(EnclaveData* enclaveData, std::string json_str);
+            WorkOrderProcessor() {}
+            ~WorkOrderProcessor() {
+                // Sanitize class members storing secrets
+                worker_encryption_key.clear();
+                session_key.clear();
```

Comment:
Can we move this implementation of destructor to  WorkOrderProcessor.cpp

Reply:
Ok